deployment/cdk/opensearch-service-migration/lib/service-stacks/migration-console-stack.ts

```diff
@@ -151,10 +151,6 @@ export class MigrationConsoleStack extends MigrationServiceCore {
             ...props,
             parameter: MigrationSSMParameter.OS_CLUSTER_ENDPOINT,
         });
-        const sourceClusterEndpoint = props.sourceClusterDisabled ? null : getMigrationStringParameterValue(this, {
```

Comment:
General observation: Whenever we make the change to use the source cluster details that are provided by the user in constructing the commands for RFS,Fetch,Capture Proxy, etc. it seems like we can remove this SSM Parameter from being needed at all

Reply:
Yes, I'm hoping so! I'll do a more comprehensive look at whether we can pull it out of all of them.